test/fixtures/integration-tests/retries/oss-rspec-failed-not-quarantined.json

```diff
@@ -0,0 +1 @@
+../rspec-failed-not-quarantined.json
```

Comment:
What's this diff mean? 😅

Reply:
These are symlinks, and I feel like we want that gitignored?

Reply:
Maybe not, we already have a bunch. Why are they there? 🤔

Reply:
I think the tests generate these if they don't already exist and then use them as config files for the various cases

Reply:
I wonder if we should gitignore them and just let the tests generate them- if retries are using them, they'll be deleted and recreated constantly anyway

Reply:
Not something you have to do in this PR though